tests/py/dynamo/conversion/test_log10.py

```diff
@@ -0,0 +1,49 @@
+import torch
+import torch.nn as nn
+from parameterized import parameterized
+from torch.testing._internal.common_utils import run_tests
+
+from .harness import DispatchTestCase
+
+
+class TestLogConverter(DispatchTestCase):
+    @parameterized.expand(
+        [
+            ((10,), torch.float),
+            ((1, 20), torch.float),
+            ((2, 3, 4), torch.float),
+            ((2, 3, 4, 5), torch.float),
+        ]
+    )
+    def test_log10_float(self, input_shape, dtype):
+        class log(nn.Module):
+            def forward(self, input):
+                return torch.log10(input)
```

Comment:
It would be more consistent with other converters to use `return torch.ops.aten.log10.default(input)`